config/main.py

```diff
@@ -470,10 +470,21 @@ def _clear_qos():
             'BUFFER_PROFILE',
             'BUFFER_PG',
             'BUFFER_QUEUE']
-    config_db = ConfigDBConnector()
-    config_db.connect()
-    for qos_table in QOS_TABLE_NAMES:
-        config_db.delete_table(qos_table)
+
+    namespace_list = [DEFAULT_NAMESPACE]
+    if sonic_device_util.get_num_npus() > 1:
+        namespace_list = sonic_device_util.get_namespaces()
+
+    for ns in namespace_list:
+        if ns is DEFAULT_NAMESPACE:
```

Comment:
This if check is not required, as the below call will work when ns is DEFAULT_NAMESPACE.
 config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=ns)

Reply:
I verified setting ns to 'DEFAULT_NAMESPACE'. It didn't work.